README.md

```diff
@@ -72,24 +72,38 @@ of the Azure Active Directory window respectively as *Name* and *Directory ID*
 
 #### Register the service app (TodoListService-NativeDotNet)
 
-1. In the  **Azure Active Directory** pane, click on **App registrations** and choose **New application registration**.
-1. Enter a friendly name for the application, for example 'TodoListService-NativeDotNet' and select 'Web app / API' as the *Application Type*.
-1. For the *Sign-on URL*, enter the base URL for the sample. By default, this sample uses `https://localhost:44321/`.
-1. Click **Create** to create the application.
-1. In the succeeding page, Find the *Application ID* value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
-1. Then click on **Settings**, and choose **Properties**.
-1. For the App ID URI, replace the guid in the generated URI 'https://\<your_tenant_name\>/\<guid\>', with the name of your service, for example, 'https://\<your_tenant_name\>/TodoListService-NativeDotNet' (replacing `<your_tenant_name>` with the name of your Azure AD tenant)
+1. In the  **Azure Active Directory** pane, click on **[App registrations](https://go.microsoft.com/fwlink/?linkid=2083908)** and choose **New registration**.
+1. Enter a friendly name for the application, for example `TodoListService-NativeDotNet`.
+1. Leave **Supported account types** on the default setting of **Accounts in this organizational directory only**.
+1. For the *Redirect URI (optional)*, select "Web" from the dropdown, and enter the base URL for the sample. By default, this sample uses `https://localhost:44321/`.
+1. Click **Register** to create the application.
+1. In the succeeding page, Find the *Application (client) ID* value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
+1. In the list of pages for the app, select on **Expose an API**
+   - Select **Add a scope**
+   - accept the proposed Application ID URI (api://{clientId}) by selecting **Save and Continue**

```

Comment:
actually no, @AllisonAm, I would not think we want to accept the proposed Application ID URI for the v1.0 samples, as otherwise we also need to change the config file.
I think you probably want to add:
- change the proposed application ID (api://{clientId}) by `https://\<your_tenant_name\>/TodoListService-NativeDotNet` (replacing `<your_tenant_name>` with the name of your Azure AD tenant)
- select **Save and Continue**

The alternative is do what you propose, but change the 5th step of the section
```md
#### Configure the client project` of the README.md
...
1. Find the app key `todo:TodoListResourceId` and replace the existing value with the 
    App ID URI you registered earlier for the TodoListService-NativeDotNet app. For instance use 
    `(api://{clientIdOfTheService}`
```

I leave it to you to decide